nrf_wifi/utils/src/util.c

```diff
@@ -80,15 +81,15 @@ int nrf_wifi_utils_hex_str_to_val(struct nrf_wifi_osal_priv *opriv,
 
 
 bool nrf_wifi_utils_is_mac_addr_valid(struct nrf_wifi_osal_priv *opriv,
-				      const char *mac_addr)
+					  const char *mac_addr)
```

Comment:
Can all these whitespace changes in this be moved to a new commit?

Reply:
Except nrf_wifi_init_ph_rf_params() function retained the old functions as is so, no need of having a separate commit.